components/PostList/index.js

```diff
@@ -12,7 +14,23 @@ import {
 } from './styles';
 import connect from './store';
 
-const PostList = ({ data, loadMorePosts }) => {
+type Post = {
```

Comment:
I'm not sure but Should we move these types declarations to another file? Like "index.types.js"?

Reply:
Well, is a matter of choosing a pattern I think, I don't put types on a separate files, unless its a very common type used thoroughly the entire app, so I usually keep them on the same file I use it, for clarity, and if I need in a couple of other files, just make it exportable.
But whatever you tell me, i'll do it, so however you prefer 😄